troposphere/static/js/components/modals/admin/ManageUserModal.react.js

```diff
@@ -0,0 +1,72 @@
+define(function (require) {
+    var React = require('react/addons'),
+        InteractiveDateField = require('components/common/InteractiveDateField.react'),
+        BootstrapModalMixin = require('components/mixins/BootstrapModalMixin.react');
+
+    
+    return React.createClass({
+
+    getInitialState:function(){
+        return {
+            endDate:""
+        }
```

Comment:
This is covered with semi-colon insertion, I believe. But I would be explicit here and add it.